### PR TITLE
MIS-41 Update node dependencies

### PR DESCRIPTION
**Pull request by MattyJ007**
To get security patches

MIS-67

### CODE REVIEW

**Comment by BMartinos** [comment]
@MattyJ007 there still seems to be some out-dated dependencies for this mediator. They are low risk, but i think it will be good to resolve and have zero issues

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "openhim-mediator-file-queue",
-  "version": "3.0.1",
+  "version": "3.0.3",
```
Comment (BMartinos) [inline package.json]: I think this was prob meant to be **3.0.2**? :)

Comment (MattyJ007) [reply package.json]: I already created a tag for 3.0.2 before your other feedback :/

Comment (BMartinos) [reply package.json]: We can just delete that tag and create a new one with the correct version in

**Pull request merged**